app/templates/application.hbs

```diff
@@ -1,3 +1,17 @@
-<h2 id="title">Welcome to Ember</h2>
+<h2 id="title">Emily's Like Counter</h2>
+
+
+
+<ul>
+  {{#each model as |movie|}}
+  <li>
+    {{movie.title}}
+<p>
+{{like-counter}}
```

Comment:
When creating new instances of `like-counter` pass in the `score` attribute from the movie so that you have a starting value.